src/pages/index.js

```diff
@@ -0,0 +1,303 @@
+document.getElementById("logo").src = logoSrc;
+document.getElementById("edit-icon").src = editIconSrc;
+document.getElementById("plus-icon").src = plusIconSrc;
+document.getElementById("close-icon-1").src = closeIconSrc;
+document.getElementById("close-icon-2").src = closeIconSrc;
+document.getElementById("close-icon-3").src = closeIconSrc;
+document.getElementById("close-icon-4").src = closeIconSrc;
+
+// API setup
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "ab8dc431-ba9b-41ea-813d-3d243375af30",
+    "Content-Type": "application/json",
+  },
+});
+
+// DOM elements
+const profileEditButton = document.querySelector(".profile__edit-btn");
+const cardModalButton = document.querySelector(".profile__add-btn");
+const profileName = document.querySelector(".profile__name");
+const profileDescription = document.querySelector(".profile__description");
+
+const editModal = document.querySelector("#edit-modal");
+const editFormElement = editModal.querySelector("#profile__form");
+const editModalCloseBtn = editModal.querySelector(".modal__close-btn");
+const editModalNameInput = editModal.querySelector("#profile-name-input");
+const editModalDescriptionInput = editModal.querySelector(
+  "#profile-description-input"
+);
+
+const cardModal = document.querySelector("#add-card-modal");
+const cardForm = cardModal.querySelector(".modal__form");
+const cardSubmitBtn = cardModal.querySelector(".modal__submit-btn");
+const cardModalCloseBtn = cardModal.querySelector(".modal__close-btn");
+const cardNameInput = cardModal.querySelector("#add-card-name-input");
+const cardLinkInput = cardModal.querySelector("#add-card-link-input");
+
+const avatarModalButton = document.querySelector(".profile__avatar-btn");
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarForm = document.querySelector("#edit-avatar-form");
+const avatarInput = avatarForm.querySelector("#profile-avatar-input");
+const avatarModalCloseBtn = avatarModal.querySelector(".modal__close-btn");
+
+const deleteModal = document.querySelector("#delete-modal");
+const deleteForm = deleteModal.querySelector(".modal__form");
+const deleteCancelBtn = deleteModal.querySelector(
+  ".modal__submit-btn:last-of-type"
+);
+
+const previewModal = document.querySelector("#preview-modal");
+const previewModalImageEl = previewModal.querySelector(".modal__image");
+const previewModalCaptionEl = previewModal.querySelector(".modal__caption");
+const previewModalCloseBtn = previewModal.querySelector(".modal__close-btn");
+
+const cardTemplate = document.querySelector("#card-template");
+const cardsList = document.querySelector(".cards__list");
+
+// Load user and cards
+api
+  .getAppInfo()
+  .then(([user, cards]) => {
+    currentUserId = user._id;
+    profileName.textContent = user.name;
+    profileDescription.textContent = user.about;
+    document.getElementById("profile-avatar").src = user.avatar;
```

Comment:
`document.getElementById("profile-avatar")`

It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine